docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+!build/
```

Comment:
Actually, I'm a bit confused if it's `docs/build/` or `docs/_build/` that you want Git to ignore.